tensornetwork/matrixproductstates/base_mps.py

```diff
@@ -53,16 +55,16 @@ class BaseMPS:
   """
 
   def __init__(self,
-               tensors: List[Union[BaseNode, Tensor]],
+               tensors: List[Tensor],
                center_position: Optional[int] = 0,
-               connector_matrix: Optional[Union[BaseNode, Tensor]] = None,
-               backend: Optional[Union[Text, BaseBackend]] = None) -> None:
+               connector_matrix: Optional[Tensor] = None,
+               backend: Optional[Text] = None) -> None:
```

Comment:
Why can't this take in a `BaseBackend` anymore?

Reply:
oops, I was over-eagerly removing Unions ...